sql-cli/examples/advanced/mart/union_top_and_last.sql

```diff
@@ -0,0 +1,3 @@
+SELECT title, rating from {{source_top_five_animations}}
```

Comment:
I see these more as test fixtures than examples, from a `sql-cli` project perspective, WDYT @feluelle ?

Reply:
But these are the same examples we have for `python-sdk`. I think it makes sense to have some examples in an `examples` directory - other than tests. For tests we can also use shorter variables such as `a`, `b`, ... 😄